Lesson34/selcan/lesson34-app/src/App.js

```diff
@@ -0,0 +1,51 @@
+import { List } from './components/List';
+import { Navbar } from './components/Navbar';
+import { PerformanceState } from './components/PerformanceState';
+import { useState } from 'react';
+import { todoData } from './data';
+import { UserProvider } from './userContext';
+import { Account } from './components/Account';
+import Login from './components/Login';
+import { useContext } from 'react';
+import { UserContext } from './userContext';
+import './App.css';
+
+// TODO: Create a new Component that contains log in form, so that user can type username and password and click submit button to log in.
+// If user is logged in, show Account component. 
+// If user is logged out, show Log In component.
+
+export const App = () => {
+	const [todos, setTodos] = useState(todoData);
+
+	const loggedOutUser = {
+		isLoggedInUser: false
+	};
+	const user = useContext(UserContext);
+
+	return (
+
+
+
+
+		<UserProvider initialState={loggedOutUser}>
+			<div className='container'>
+				<Navbar />
+				<div>
```

Comment:
Since this div element is not really used for anything apart from complying with a rule of single parent, replace it with fragment component:
```suggestion
				<>
```